src/chapters/databases-part2/db-setup.rst

```diff
@@ -0,0 +1,101 @@
+   value as well as a location (row letter plus column number).
+#. Information about each individual cell on the board. This includes a primary
+   key, the cell's location, the number of mines in surrounding cells, whether
+   the user has clicked the cell, and whether the cell contains a mine.
+
+This outline suggests at 2 tables in the database - one for a list of mines,
+and another for the cells on the game board.
+
+``db_setup.py``
+---------------
+
+Use the tools in Visual Studio code to create a new Python file at the same
+level as ``main.py``. Call the new file ``db_setup.py``. This will be a
+separate program, and its only job is to create the database file and add two
+tables.
+
+Follow along with the video to build this program.
+
+.. admonition:: Tip
+
+   The video player automatically sets the resolution. However, this choice
+   might not be good enough to clearly see all of the text.
+
+   If the code looks blurry on your screen, click the gear icon in the video
+   controls and increase the resolution to the maximum value.
+   
+.. raw:: html
+
+   <section class="vid_box">
+      <iframe class="vid" src="https://www.youtube.com/embed/arIdYX7dBDU" frameborder="1" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+   </section>
+
+.. admonition:: Note
+
+   For our basic game, we might not use all of the columns in the tables yet.
+   However, they could be useful later, so there's no harm in adding them now.
+
+Video Summary
+-------------
+
+.. admonition:: Tip
+
+   Don't try to code ``db_setup.py`` using only the summary! The clip contains
+   information and specific code instructions that you really shouldn't skip.
+
+   The text by itself doesn't provide enough detail on its own. It's a
+   *SUMMARY* and not a *GMESD* (Give-Me-Every-Single-Detail).
+
+#. After creating the ``game.db`` database file, add two new tables: ``mines``
+   and ``board``. The first table stores an ID number and location for each
+   mine created in the game. The second table stores information about each
+   cell on the game board.
+#. Adding ``DROP TABLE`` commands at the top of the file lets us quickly remove
+   and recreate the two tables. This is useful if we need to make a fresh
+   start.
+#. The ``mines`` table includes two columns: ``mine_id`` and ``coordinates``.
+   ``mine_id`` is the primary key, and its data type must be ``INTEGER`` instead
+   of ``INT``. (For some reason, using the ``INT`` syntax doesn't automatically
+   increment the primary key values).
+
+   ``coordinates`` is of type ``TEXT`` and will store values like ``C9`` or
+   ``E2``.
+#. The ``board`` table includes columns for ``cell_id`` (the primary key),
+   ``coordinates``, ``surr_mines`` (surrounding mines), ``guessed`` and
+   ``mine_id``.
+
+   ``cell_id``, ``surr_mines``, and ``mine_id`` are integers. ``coordinates``
+   is a text data type, and ``guessed`` is a boolean.
```

Comment:
Crud.  I should have used `BOOL` for `guessed` in the `sql_query` string.

`sqlite3` doesn't actually have a boolean data type, so T/F automatically get converted to the integers 0 and 1.  That's what happened here, although TEXT (maybe) instead of INT.

I'm going to see if I can overlay the word BOOL over TEXT in the clip.  That might save me from having to redo the whole thing!

Reply:
I wound up re-recording the video.  It's 1:10 min longer than the original, but it properly sets `guessed` as a boolean data type now.